Various frontend QoL fixes

e32wong opened the pull request:
UI Changes:
- Move UI to a temporary drawer layout before we remove all of Quasar's layout system
- Updated opinion highlighting to highlight the entire opinion block instead of just the opinion body
- Set the cluster colors to be all blue when "all" tab is selected
- Use DiceBear instead of Boring Avatars for user profile pictures

Fixes:
- Back button now properly includes the route query and params for the Vue router
- Deleting an opinion will no longer trigger a zod error on the opinion count
- Revised the conversation page to track the filter algorithms internally instead of relying on the URL params

Features:
- Added link support in HTML bodies for conversations and opinions

nicobao commented:
This is how it shows when the user is not logged-in: 
![image](https://github.com/user-attachments/assets/1aca3ddb-2076-47a8-b05d-67ec378b06f7)

Change this to:
- remove the icon:
![image](https://github.com/user-attachments/assets/64a286a7-5740-46be-8f72-768e1c7589e8)
- put "Log in" button on the left (where is currently the removed icon)

nicobao commented:
This is how it shows when the user is logged-in:
![image](https://github.com/user-attachments/assets/0bc8896a-1f10-46b7-bfa9-915d4596f710)

Change as follows:
- replace the symbol ![image](https://github.com/user-attachments/assets/64a286a7-5740-46be-8f72-768e1c7589e8) with icon of the user ![image](https://github.com/user-attachments/assets/dc810d40-1c15-4a09-a29c-88faa75ff70a)

nicobao commented:
For both states, remove this separation line between the top bar and the rest of the feed:
![image](https://github.com/user-attachments/assets/d5a29791-d4a6-4fd9-b87e-3300d32c7122)

nicobao commented:
This is a conversation page looks:
![image](https://github.com/user-attachments/assets/302ccf41-892e-4be7-abcb-65e9c195d421)

Change the top bar to the following:
- ![image](https://github.com/user-attachments/assets/b8f75aa2-621b-48e2-a5a1-d4e88aaed870)
- in other words, remove the drawer icon (or its profile picture replacement)

nicobao commented:
The profile page currently looks like:
![image](https://github.com/user-attachments/assets/96e2810c-a93e-4e4f-b4fe-ccdb552dcc07)

Instead, change as follows:
- remove the top "profile pic" that opens the drawer
- replace it with the back button
- remove the bottom navbar

nicobao commented:
Could you also change the "green" colors in the profile page "Conversation/Opinion" tabs to the gradient-blue/purple color like in the Figma?:
![image](https://github.com/user-attachments/assets/c38bc772-d8f9-43cf-ab19-8b4f3a1f9b53)

Same for the "blue" color of the bottom navbar:
- before: ![image](https://github.com/user-attachments/assets/80fbb3a9-87e4-4af4-9f68-7d6c6398c6c7)
- after: ![image](https://github.com/user-attachments/assets/4f62429a-6b3d-490f-9ade-2a23fd2ffc67)

Same for the "static purple color" of the "add a conversation" button:
- before: ![image](https://github.com/user-attachments/assets/461042c3-0037-4bd3-b094-4208f2d78e7a)
- after: ![image](https://github.com/user-attachments/assets/8ec673eb-5cf7-4d10-b233-c5cc28a8b10b)

Worth creating a proper quasar color for this https://quasar.dev/style/color-palette/ to reuse across the app (we need two variables with or without the gradient, to apply depending on the UI element)

nicobao commented:
Could you also replace the "share" icon to match the Figma https://www.figma.com/design/AD1cYjYTAz47YRInLqrLAV/Agora-Citizen-Network-Prototype?node-id=17538-53391&t=NJKVOeiFMPGrvMMa-0:
- before: ![image](https://github.com/user-attachments/assets/d744b90f-c4d2-4e7a-a7eb-ab6f208995d2)
- after: ![image](https://github.com/user-attachments/assets/189ea81b-8baa-4e2e-a728-df710590dbe9)

nicobao commented:
Fix the two vertical scrolling on the main feed (keep only the most external one I suppose)
![image](https://github.com/user-attachments/assets/4211e77e-5ecd-4fe5-b020-64d3e3762bf6)

nicobao commented:
Add the logo in the middle of the top navbar, only in the main feed:

![image](https://github.com/user-attachments/assets/7dbac8a1-0662-4042-806e-6d135bb43932)

![image](https://github.com/user-attachments/assets/c990acc8-6341-4efe-9ec9-e0123ca4de11)

On click on the topbar, scroll to top.

nicobao commented:
In the main feed, scroll to top (smoothly):
- on click on the "Home button"
- on click on anywhere in the top navbar, except for the "profile" button (like on Twitter)

nicobao commented:
Could you help me highlight correctly the <a> links ("terms of service" and "privacy policy") in the following: 
- login page: ![image](https://github.com/user-attachments/assets/0635959b-11b0-4bd2-9a92-955243828126)
- registration page: ![image](https://github.com/user-attachments/assets/f2a46982-887d-4b6c-9ee1-0e65e3f3b77f)

(was my own addition)

We'll keep the two buttons, but on the backend side I'll return a value that says whether it was a log in or a register, so you know which page to follow-up with.

nicobao commented:
In the  passport verification page, is it possible to change the grey color to white?

![image](https://github.com/user-attachments/assets/6a5b12dc-6235-4d11-a71b-d1e1f51d566a)

nicobao commented:
Is it also possible to use these icons?
https://www.figma.com/design/AD1cYjYTAz47YRInLqrLAV/Agora-Citizen-Network-Prototype?node-id=784-6801&p=f&t=YGVbmqEsElCx3uwk-0
To choose the icon, George used https://icon-sets.iconify.design/ (that's what the string above the icon means!)
Let me know if you face any issues when integrating with the icons

nicobao commented:
> For both states, remove this separation line between the top bar and the rest of the feed: ![image](https://private-user-images.githubusercontent.com/46368291/416012717-e7762dd1-d62f-4c20-a9d1-792f852b0043.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDA1MjU5MTQsIm5iZiI6MTc0MDUyNTYxNCwicGF0aCI6Ii80NjM2ODI5MS80MTYwMTI3MTctZTc3NjJkZDEtZDYyZi00YzIwLWE5ZDEtNzkyZjg1MmIwMDQzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjI1VDIzMjAxNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWMyMGZkYTc4OGZkNTdmYmJiMTY3MzA4OTU1Y2EyNDAwYTVhYWFiOTIwNzY1ZDU4NjA4M2MxZDlkYWJmZjFhYjkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.KxDaFQJ8T8QzGTvdsiGgdpwv2oGIF9Y-ttdTdNzjMyQ)

Actually, we should keep, just make it more discrete. Otherwise it's not clear when scrolling, since we're not removing the top navbar as we scroll anymore (that was provided by q-header and q-footer, and I would like to retain this behavior, but it's not priority)...

nicobao commented:
Make the edges of the profile picture more rounded:
- before: ![image](https://github.com/user-attachments/assets/d8c274da-1b39-45f4-a2c8-6f2d4d094d5a)
- after: ![image](https://github.com/user-attachments/assets/1b51f7f5-40e7-42f4-8873-f6d13fe0456c)

e32wong commented:
> This is how it shows when the user is not logged-in: ![image](https://private-user-images.githubusercontent.com/46368291/416012417-1aca3ddb-2076-47a8-b05d-67ec378b06f7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDA3ODU2NjcsIm5iZiI6MTc0MDc4NTM2NywicGF0aCI6Ii80NjM2ODI5MS80MTYwMTI0MTctMWFjYTNkZGItMjA3Ni00N2E4LWIwNWQtNjdlYzM3OGIwNmY3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjI4VDIzMjkyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFjMTU5NGRkMDFhY2Y1Yzc4N2FhMDg2YmIxNTk2OGI2OThhMjE2MWU0ZDk0ODg5OWM2Y2ZmZWI5OTAyMjUzMmYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.jCEJShUPD08_OkcgOK6qRLGvwHdAaPT3a1auImZG9Pc)
> 
> Change this to:
> 
>     * remove the icon:
>       ![image](https://private-user-images.githubusercontent.com/46368291/416012495-64a286a7-5740-46be-8f72-768e1c7589e8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDA3ODU2NjcsIm5iZiI6MTc0MDc4NTM2NywicGF0aCI6Ii80NjM2ODI5MS80MTYwMTI0OTUtNjRhMjg2YTctNTc0MC00NmJlLThmNzItNzY4ZTFjNzU4OWU4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjI4VDIzMjkyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTJkNzZhZDFkZmU0ZTNmMDZiYWY4YjcwZGU3ZWI4YjlkYTQzYjZiYWM3ZjAzZGIxNTM5NDQzNjI2YjRlYTI0OWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.CHgQGx6U1_iHosBDMFsMktnSVkMUbcahlsv0Rz_HTmo)
> 
>     * put "Log in" button on the left (where is currently the removed icon)

Ignored this as per discussion.

e32wong commented:
> This is how it shows when the user is logged-in: ![image](https://private-user-images.githubusercontent.com/46368291/416012546-0bc8896a-1f10-46b7-bfa9-915d4596f710.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDA3ODU2NjcsIm5iZiI6MTc0MDc4NTM2NywicGF0aCI6Ii80NjM2ODI5MS80MTYwMTI1NDYtMGJjODg5NmEtMWYxMC00NmI3LWJmYTktOTE1ZDQ1OTZmNzEwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjI4VDIzMjkyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTM4MjE1YzUxZTc5OTI3ZWMwNDhjNDEwMTQxNTdhZTJlMDBmNzYxMjZiMWExZThmNDA3ZWUyOTU1NTc1NDQwODMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.n4TWcyUpxfKC8dSrQABlNFCFQq7cOzmZRsTfTNfRB58)
> 
> Change as follows:
> 
>     * replace the symbol ![image](https://private-user-images.githubusercontent.com/46368291/416012495-64a286a7-5740-46be-8f72-768e1c7589e8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDA3ODU2NjcsIm5iZiI6MTc0MDc4NTM2NywicGF0aCI6Ii80NjM2ODI5MS80MTYwMTI0OTUtNjRhMjg2YTctNTc0MC00NmJlLThmNzItNzY4ZTFjNzU4OWU4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjI4VDIzMjkyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTJkNzZhZDFkZmU0ZTNmMDZiYWY4YjcwZGU3ZWI4YjlkYTQzYjZiYWM3ZjAzZGIxNTM5NDQzNjI2YjRlYTI0OWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.CHgQGx6U1_iHosBDMFsMktnSVkMUbcahlsv0Rz_HTmo) with icon of the user ![image](https://private-user-images.githubusercontent.com/46368291/416012604-dc810d40-1c15-4a09-a29c-88faa75ff70a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDA3ODU2NjcsIm5iZiI6MTc0MDc4NTM2NywicGF0aCI6Ii80NjM2ODI5MS80MTYwMTI2MDQtZGM4MTBkNDAtMWMxNS00YTA5LWEyOWMtODhmYWE3NWZmNzBhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjI4VDIzMjkyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWRkZjE5NDBmYmUzYTI4YTM2N2M5MjJmMmIxN2YyODZiNWE5Zjc5YWI0Y2JlNmJjOWRmZWQ5ZDAyOTc3MjlkZjkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.tDjLgqEqbxJsZq5aRtJE9JMqNM7Lwfzy3TJZVn2Ysy0)

Updated as per recommendation.

e32wong commented:
> For both states, remove this separation line between the top bar and the rest of the feed: ![image](https://private-user-images.githubusercontent.com/46368291/416012717-e7762dd1-d62f-4c20-a9d1-792f852b0043.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDA3ODU2NjcsIm5iZiI6MTc0MDc4NTM2NywicGF0aCI6Ii80NjM2ODI5MS80MTYwMTI3MTctZTc3NjJkZDEtZDYyZi00YzIwLWE5ZDEtNzkyZjg1MmIwMDQzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjI4VDIzMjkyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI4Y2ZiNWYwOGM3Nzc3ZTdkNGM4YzVlMzg0NjVmNzYxNzNiY2YzZTM1YzgzYTc1MTU4NTkzZjE4NTg2NjFlYzAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.bxmZ0hjF-mLIWPC-02GfdZmGw9pgd0OBLnKZRGS9lqU)

I have enabled header's "reveal mode" so this change had been ignored.

e32wong commented:
> This is a conversation page looks: ![image](https://private-user-images.githubusercontent.com/46368291/416012779-302ccf41-892e-4be7-abcb-65e9c195d421.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDA3ODU2NjcsIm5iZiI6MTc0MDc4NTM2NywicGF0aCI6Ii80NjM2ODI5MS80MTYwMTI3NzktMzAyY2NmNDEtODkyZS00YmU3LWFiY2ItNjVlOWMxOTVkNDIxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjI4VDIzMjkyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTlmOGM0YzY2YTI5MmI3YTE4NzYyMTliYjJmOWYwMzU1MDllN2M5M2RkMWUxMjNjNzAwYTYzYWI0YzIxNjUxYWYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.weYkZf-JZ9nw8GfBZsTa_Ybd1O2rszSF-qdzAbEi5hg)
> 
> Change the top bar to the following:
> 
>     * ![image](https://private-user-images.githubusercontent.com/46368291/416012932-b8f75aa2-621b-48e2-a5a1-d4e88aaed870.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDA3ODU2NjcsIm5iZiI6MTc0MDc4NTM2NywicGF0aCI6Ii80NjM2ODI5MS80MTYwMTI5MzItYjhmNzVhYTItNjIxYi00OGUyLWE1YTEtZDRlODhhYWVkODcwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjI4VDIzMjkyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTAzNGNiYjg5Yjk2YjhjYmE4YjQ4NTk4NDg1YzcyNzQyNWQyYjJjNjcxNWNjYjY4M2Q1M2Q1NDgwNWJiNzk4NTAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.m4lUJuBwkWykvk1MsjL491okktHV76H8o7RY7xBrGRk)
> 
>     * in other words, replace the drawer icon (or its profile picture replacement) with the back arrow and remove the drawer entirely

Updated as per recommendation. The header now only contains a back arrow.

e32wong commented:
> The profile page currently looks like: ![image](https://private-user-images.githubusercontent.com/46368291/416014457-96e2810c-a93e-4e4f-b4fe-ccdb552dcc07.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDA3ODU2NjcsIm5iZiI6MTc0MDc4NTM2NywicGF0aCI6Ii80NjM2ODI5MS80MTYwMTQ0NTctOTZlMjgxMGMtYTkzZS00ZTRmLWI0ZmUtY2NkYjU1MmRjYzA3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjI4VDIzMjkyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWVhY2M5NTEyMzY0NGVkZDEyMDFjMTA2YzljYjJhMDA2ZTlhMDRjY2I4Y2QyNmY1NTMyNWI2OWEyNzkwYWEzNmYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.ztSZTWZoJRSz2cftbviUPx5rczckp0yOn6ujkk0zOZc)
> 
> Instead, change as follows:
> 
>     * remove the top "profile pic" that opens the drawer
> 
>     * replace it with the back button
> 
>     * remove the bottom navbar
> 
> 
> So it would look like in Figma: https://www.figma.com/design/AD1cYjYTAz47YRInLqrLAV/Agora-Citizen-Network-Prototype?node-id=17538-56810&t=NJKVOeiFMPGrvMMa-0

User profile had been updated as per recommendation.

e32wong commented:
> Could you also change the "green" colors in the profile page "Conversation/Opinion" tabs to the gradient-blue/purple color like in the Figma?:
> 
>     * before: ![image](https://private-user-images.githubusercontent.com/46368291/416014943-0cb562dc-70e5-4f3f-a77e-8b690732af44.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDA3ODU2NjcsIm5iZiI6MTc0MDc4NTM2NywicGF0aCI6Ii80NjM2ODI5MS80MTYwMTQ5NDMtMGNiNTYyZGMtNzBlNS00ZjNmLWE3N2UtOGI2OTA3MzJhZjQ0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjI4VDIzMjkyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTg4YmQzYjQ0NTk4Y2VlMDJlODEyNjdmNGRkZDJhYTBiM2VhYjI5NTM1ZWJkYWVkODA1ODJhYmYzN2MxZTlhYWYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.M5iEKUN04eCk_m91wwu814zheofyr7JJGtrug1QOehI)
> 
>     * after: ![image](https://private-user-images.githubusercontent.com/46368291/416014638-c38bc772-d8f9-43cf-ab19-8b4f3a1f9b53.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDA3ODU2NjcsIm5iZiI6MTc0MDc4NTM2NywicGF0aCI6Ii80NjM2ODI5MS80MTYwMTQ2MzgtYzM4YmM3NzItZDhmOS00M2NmLWFiMTktOGI0ZjNhMWY5YjUzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjI4VDIzMjkyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI4ZGZiOGQ4MWY0ZjcxODkxMWQ3NGRkNzdiMzJhOTE4ZDRjMzUxNjg4Yjg2Nzk2OTk2NzZlZDJiN2M3NzQ5MjQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.Xcho0OVeiW5lGSu3xPsriHWU3IzMHprinJFsajnLOZI)
> 
> 
> Same for the "blue" color of the bottom navbar:
> 
>     * before: ![image](https://private-user-images.githubusercontent.com/46368291/416014723-80fbb3a9-87e4-4af4-9f68-7d6c6398c6c7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDA3ODU2NjcsIm5iZiI6MTc0MDc4NTM2NywicGF0aCI6Ii80NjM2ODI5MS80MTYwMTQ3MjMtODBmYmIzYTktODdlNC00YWY0LTlmNjgtN2Q2YzYzOThjNmM3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjI4VDIzMjkyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWVhNWNhZjg2MDc2NGQxMTFlM2Y3ZGNjYTBkZmE3MDQyYzE3MzdjYWYxOWVlY2ZkOTljMjMwZTAwMWQzZWUzOWQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.v2KjKSnjjvma4gQRiTWKmcnJOMWZLnUze09zvbyFusQ)
> 
>     * after: ![image](https://private-user-images.githubusercontent.com/46368291/416014689-4f62429a-6b3d-490f-9ade-2a23fd2ffc67.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDA3ODU2NjcsIm5iZiI6MTc0MDc4NTM2NywicGF0aCI6Ii80NjM2ODI5MS80MTYwMTQ2ODktNGY2MjQyOWEtNmIzZC00OTBmLTlhZGUtMmEyM2ZkMmZmYzY3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjI4VDIzMjkyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM5ODRlYzM3MTZkMjYzZTY3Zjc1ODUwYjkxZDQyYjliNjMyMjQ0MmU5NWJlYzQ3Y2QwYjJlMDk4OTk3ZmI0ZDMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.fbLUASt_jvn4knVX5YqAPyOO_GjTRNfB0ri0LQnwWJQ)
> 
> 
> Same for the "static purple color" of the "add a conversation" button:
> 
>     * before: ![image](https://private-user-images.githubusercontent.com/46368291/416014750-461042c3-0037-4bd3-b094-4208f2d78e7a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDA3ODU2NjcsIm5iZiI6MTc0MDc4NTM2NywicGF0aCI6Ii80NjM2ODI5MS80MTYwMTQ3NTAtNDYxMDQyYzMtMDAzNy00YmQzLWIwOTQtNDIwOGYyZDc4ZTdhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjI4VDIzMjkyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI2MDY0NjRlMWYzOWIwN2JkMGIxYzRjYzA3Nzk4ZTUxYWNiMTQ3MzllMjA1YjFhOWRjODkzZDBmZmJiNjVkOTkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.5xmeofRGlrvnNsJHX6IauBBIJU2CdiEJbUiiXhpueBY)
> 
>     * after: ![image](https://private-user-images.githubusercontent.com/46368291/416014773-8ec673eb-5cf7-4d10-b233-c5cc28a8b10b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDA3ODU2NjcsIm5iZiI6MTc0MDc4NTM2NywicGF0aCI6Ii80NjM2ODI5MS80MTYwMTQ3NzMtOGVjNjczZWItNWNmNy00ZDEwLWIyMzMtYzVjYzI4YThiMTBiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjI4VDIzMjkyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTFjMzc0OTdiNTMxMWFjZTI4OTM0OGRlNDkxNTNmN2IzZTczYWUxMjgzODMzOTdjNDM4OTIyNTY4N2ZhNWRiNzImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.G8Z9wHlyRJqGKozYmrFLkoqR_9o-sw4bWj38fJTa5zg)
> 
> 
> Worth creating a proper quasar color for this https://quasar.dev/style/color-palette/ to reuse across the app (we need two variables with or without the gradient, to apply depending on the UI element)

- Updated the profile and conversation tabs with the gradient color
- Updated the footer with the gradient color
- New conversation button now uses the gradient color

e32wong commented:
> Could you also replace the "share" icon to match the Figma https://www.figma.com/design/AD1cYjYTAz47YRInLqrLAV/Agora-Citizen-Network-Prototype?node-id=17538-53391&t=NJKVOeiFMPGrvMMa-0:
> 
>     * before: ![image](https://private-user-images.githubusercontent.com/46368291/416015121-d744b90f-c4d2-4e7a-a7eb-ab6f208995d2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDA3ODU2NjcsIm5iZiI6MTc0MDc4NTM2NywicGF0aCI6Ii80NjM2ODI5MS80MTYwMTUxMjEtZDc0NGI5MGYtYzRkMi00ZTdhLWE3ZWItYWI2ZjIwODk5NWQyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjI4VDIzMjkyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ4YmI0MWZiMWEzNDJjYmViOTg1YjZmNDhmZmI0ZmQxYzUwNzEzZjhkZmM3MDE3ODI1ZDNlYTBmNjliYmZmZWMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.Psb6jbA_el_9eEeAn1ypjt4Bd4LlkPpVmPkOGLjPVXs)
> 
>     * after: ![image](https://private-user-images.githubusercontent.com/46368291/416015174-9b67bcf2-9361-4ed1-9e13-dfe7a43f2fad.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDA3ODU2NjcsIm5iZiI6MTc0MDc4NTM2NywicGF0aCI6Ii80NjM2ODI5MS80MTYwMTUxNzQtOWI2N2JjZjItOTM2MS00ZWQxLTllMTMtZGZlN2E0M2YyZmFkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjI4VDIzMjkyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFkMzk1ZmY2ZDRhNThkMTA5YjIzZjFmNGU1YmYxNzhlMjhkYzc0NWRlNzEzNTBlOGRjOGQ2ZTQ2NzkwOTIwM2MmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.kV9JynCBcYXOyvgY55fKBO1xzRCWmVkXRdEDSISBgms)

Share button had been updated as per recommendation.

e32wong commented:
> Fix the two vertical scrolling on the main feed (keep only the most external one I suppose) ![image](https://private-user-images.githubusercontent.com/46368291/416015397-4211e77e-5ecd-4fe5-b020-64d3e3762bf6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDA3ODU2NjcsIm5iZiI6MTc0MDc4NTM2NywicGF0aCI6Ii80NjM2ODI5MS80MTYwMTUzOTctNDIxMWU3N2UtNWVjZC00ZmU1LWIwMjAtNjRkM2UzNzYyYmY2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjI4VDIzMjkyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWRjNTFkZWEyMTQ4MmNhMGNhZDFkZTBmZDgwMjI4YzBmOWJkZDI1NWNiZjliZDQ4N2IyOWJkNjQ5YzFjNGNjMGUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.Ahwn2piiqLo2BB2wrKHwvwcENj_e1npi1pC_1JJACrE)

The double scrollbar had been removed.

e32wong commented:
> Add the logo in the middle of the top navbar, only in the main feed:
> 
> ![image](https://private-user-images.githubusercontent.com/46368291/416015467-7dbac8a1-0662-4042-806e-6d135bb43932.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDA3ODU2NjcsIm5iZiI6MTc0MDc4NTM2NywicGF0aCI6Ii80NjM2ODI5MS80MTYwMTU0NjctN2RiYWM4YTEtMDY2Mi00MDQyLTgwNmUtNmQxMzViYjQzOTMyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjI4VDIzMjkyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ0OGI3M2EzYzg1OGNlYmUwYTY2NmY3MzFhOGVkMzYwMDM3YTFiY2U3ZWE5MDU1ZmUxODA4YzAyNWI0MzliNjQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.oC5oxgK27PPePEAB6NZn3Z6R0hkXnHVovwi9O49r-3I)
> 
> Like so: ![image](https://private-user-images.githubusercontent.com/46368291/416015786-67737a6d-6150-4791-a7eb-1d4034cbb49e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDA3ODU2NjcsIm5iZiI6MTc0MDc4NTM2NywicGF0aCI6Ii80NjM2ODI5MS80MTYwMTU3ODYtNjc3MzdhNmQtNjE1MC00NzkxLWE3ZWItMWQ0MDM0Y2JiNDllLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjI4VDIzMjkyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWVlYzQwODI3NTM3N2E4MGNiMTlmOTQ1M2IzOWYxYWY3Y2IwOGRiZDcwZjVlOGE3MWIwN2ZjODdjODFkYzZlMmUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.1PU9ZF29_3Y1hI9-mYHIfPVwY_BiX9YkzpCmiTos33Y)
> 
> (Do not add the "search icon" for now.)

The center logo had been added to the main feed's header bar when the user is browsing in mobile mode.

e32wong commented:
> In the main feed, scroll to top (smoothly):
> 
>     * on click on the "Home button"
> 
>     * on click on anywhere in the top navbar, except for the "profile" button (like on Twitter)

Auto scroll to top had been enabled.

e32wong commented:
> Could you help me highlight correctly the links ("terms of service" and "privacy policy") in the following:
> 
>     * login page: ![image](https://private-user-images.githubusercontent.com/46368291/416016782-0635959b-11b0-4bd2-9a92-955243828126.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDA3ODU2NjcsIm5iZiI6MTc0MDc4NTM2NywicGF0aCI6Ii80NjM2ODI5MS80MTYwMTY3ODItMDYzNTk1OWItMTFiMC00YmQyLTlhOTItOTU1MjQzODI4MTI2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjI4VDIzMjkyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU5YTk1ZmRjZTg2NDgyYTFmYWIyOWE4ZjQ2YjRlNWQ4YWZkNjIzZTk1ZDVhY2UwMGQ3N2RhNDQwNjcyNjYyOTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.HyGNQziHi1qlzXSHwnz6bZbSAGKd77lmHZi4nnB5pLU)
> 
>     * registration page: ![image](https://private-user-images.githubusercontent.com/46368291/416016821-f2a46982-887d-4b6c-9ee1-0e65e3f3b77f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDA3ODU2NjcsIm5iZiI6MTc0MDc4NTM2NywicGF0aCI6Ii80NjM2ODI5MS80MTYwMTY4MjEtZjJhNDY5ODItODg3ZC00YjZjLTllZTEtMGU2NWUzZjNiNzdmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjI4VDIzMjkyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRjNzUxZmE3YTFlNjNhNDM4MzMzMThlN2ExODE5ZjczY2I0NmFhMDUwOTVkNWRhMTRmM2JjZGU5ZTAwY2RiOGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.UfL0AGsyAgU0eZ3Blvu1hG841r8sXaS066pN97L2g1Q)
> 
> 
> (was my own addition)
> 
> We'll keep the two buttons, but on the backend side I'll return a value that says whether it was a log in or a register, so you'll know which page to follow-up with.

Restyled the href link as per recommendation.

e32wong commented:
> In the passport verification page, is it possible to change the grey color of the sides to white?
> 
> ![image](https://private-user-images.githubusercontent.com/46368291/416018375-6a5b12dc-6235-4d11-a71b-d1e1f51d566a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDA3ODU2NjcsIm5iZiI6MTc0MDc4NTM2NywicGF0aCI6Ii80NjM2ODI5MS80MTYwMTgzNzUtNmE1YjEyZGMtNjIzNS00ZDExLWE3MWItZDFlMWY1MWQ1NjZhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjI4VDIzMjkyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTczZmRjNzBiY2ExOWNhYzVjYTA3Yjk5MmNjM2Y0OWM4YzZhMzczZmVkM2Y4YTZjMDJkZGIwYmY3ZGU5MDQ3MzQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.Y6VbpDkFq70qPSPWv8svVJXyEa8N5PfVOv1MENUH-nI)

Resolved by making the bottom overlay scrollable.

e32wong commented:
> Is it also possible to use these icons? https://www.figma.com/design/AD1cYjYTAz47YRInLqrLAV/Agora-Citizen-Network-Prototype?node-id=784-6801&p=f&t=YGVbmqEsElCx3uwk-0 To choose the icon, George used https://icon-sets.iconify.design/ (that's what the string above the icon means!) Let me know if you face any issues when integrating with the icons

Added Iconify icons to the project.

e32wong commented:
> Make the edges of the profile picture more rounded:
> 
>     * before: ![image](https://private-user-images.githubusercontent.com/46368291/416911308-d8c274da-1b39-45f4-a2c8-6f2d4d094d5a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDA3ODU2NjcsIm5iZiI6MTc0MDc4NTM2NywicGF0aCI6Ii80NjM2ODI5MS80MTY5MTEzMDgtZDhjMjc0ZGEtMWIzOS00NWY0LWEyYzgtNmYyZDRkMDk0ZDVhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjI4VDIzMjkyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWZiY2MzYzRiYmI4ZjE1YTVjZjMzYzRiYjRiNzIzYzc2M2Y5MGVmNmI3NjhjYWI4NGE2ZGVkZmU5NWRmYjE0MjUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.341Vn-qjiIfMgbB-d0fiyMp00sEXw72LGXCdfDUUyTs)
> 
>     * after: ![image](https://private-user-images.githubusercontent.com/46368291/416911422-1b51f7f5-40e7-42f4-8873-f6d13fe0456c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDA3ODU2NjcsIm5iZiI6MTc0MDc4NTM2NywicGF0aCI6Ii80NjM2ODI5MS80MTY5MTE0MjItMWI1MWY3ZjUtNDBlNy00MmY0LTg4NzMtZjZkMTNmZTA0NTZjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjI4VDIzMjkyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE2N2Y4NGRmZWQxOTQ3MWZkOWIzNDcwZTM3YmM2MjNkYzgyYWNhMGVlN2NmNmY4ZWI4NmZmZGRiMGY4OTBkY2MmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.rhdsefK7tUS2ZdpySb_xXbJkEl_WLU3iAjHvrJHcBkw)

Profile pictures are now more rounded.
